internal/doc/modular_adoc_docs.md

```diff
@@ -1,4 +1,4 @@
-``# Generating AsciiDoc Docs For Your Own cobra.Command
```

Comment:
Do you mind excluding those changes as well. 

There is #1352 that addresses that